R/calibrate.R

```diff
@@ -9,5 +10,14 @@ run_calibrate <- function(model_output, calibration_options, language = NULL) {
     })
   }
 
-  naomi::hintr_calibrate(model_output, calibration_options)
+  path_results <- normalizePath(path_results, mustWork = TRUE)
+  output_path <- tempfile(tmpdir = path_results, fileext = ".rds")
```

Comment:
To save yourself a headache here I'd add a prefix.

```
tempfile("output", path_results, ".rds")
```

which and similarly for the other output types

Reply:
Thanks, have prefixed those here an in model outputs